srcpkgs/bootterm/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'bootterm'
+pkgname=bootterm
+version=0.5
+revision=1
+build_style=gnu-makefile
+# To overcome the ld linker getting handled gcc parameters (-Wl,-z,relro)
+make_use_env=yes
+# Force termios.h usage instead of asm/termbits.h (for tcgetattr, etc.)
+make_build_args="CFLAGS=-DNO_TCGETS2"
+short_desc="Simple terminal designed to ease connection to serial ports"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
Starting a package orphaned is bad form, we aren't likely to accept it if you don't want to take basic responsibility.

Reply:
@tranzystorekk I've since put `[WIP]` in the issue title, to express the state of the patch, but maybe I should use `[RFC]` instead, WDYT ?

Reply:
No need, I just wanted to make it clear that without a proper contact in the maintainer filed this is a non-starter.